platformio/commands/account/helpers.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2014-present PlatformIO <contact@platformio.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=unused-argument
+
+import os
+
+import jwt
+import requests
+
+from platformio import app, exception
+from platformio.commands.account import (
+    PIO_ACCOUNT_LOGIN_URL,
+    PIO_ACCOUNT_REFRESH_TOKEN_URL,
+)
+
```

Comment:
Implement auto-retry session for account endpoint. See https://www.peterbe.com/plog/best-practice-with-retries-with-requests.

Also, please read official docs.

The case. Something going with a network, we restart account broker, etc. Normally, we should retry within 15sec